fiftyone/utils/qwen3_vl.py

```
@@ -468,6 +468,53 @@ def embed_all(self, args):
+            a 1D numpy array embedding
+        """
+        frames = list(frames)
+        if not frames:
+            raise ValueError("Cannot embed an empty list of frames")
+
+        sample_fps = self.config.video_fps
+
+        if fps and sample_fps > 0:
+            step = max(1, round(fps / sample_fps))
+        else:
+            step = 1
```

Ruff (0.15.18)

[warning] Use ternary operator `step = max(1, round(fps / sample_fps)) if fps and sample_fps > 0 else 1` instead of `if`-`else`-block

Replace `if`-`else`-block with `step = max(1, round(fps / sample_fps)) if fps and sample_fps > 0 else 1`

(SIM108)